lib/service-data/service-data.js

```diff
@@ -30,14 +30,30 @@ const external = {}
 
 external.getRuntimeData = getRuntimeData
 
-external.loadServiceData = () => {
+external.loadServiceData = async () => {
   const sourceObjs = external.getServiceSources()
   const schemas = external.getServiceSchemas()
-  return external.getRuntimeData(sourceObjs, schemas)
-    .then(serviceData => {
-      external.setServiceInstances(serviceData)
-      return serviceData
-    })
+
+  // temporary kludge to ensure renamed models are handled
```

Comment:
Just for my benefit, what is happening here?

Reply:
The property name `model` is thoroughly misleading - should have done this ages ago, but as I was actually updating the documentation on this, it was now or never